### PR TITLE
feat(moderation): {OSL-166] Add new "Moderation" tool to the suite

### PR DESCRIPTION
**Pull request by nina-py**
## Goal

- Added routing, menu, and page scaffolding to accommodate the new "Moderation" tool to Curation Admin Tools.

- Note while I added menu links and links to the intro on the landing page for the Moderation tool, I didn't actually add components for these pages so an attempt to follow the links will end up in a 404 for now, until they're added in subsequent tasks.

##  Video walkthrough

https://user-images.githubusercontent.com/22447785/222998578-d4603653-100a-4b5f-990b-097e1a6a31de.mov

## Reference

Tickets:

- https://getpocket.atlassian.net/browse/OSL-166

### CODE REVIEW

File: src/_shared/hooks/useMozillaAuth/useMozillaAuth.ts [hunk 1]
```diff
@@ -46,8 +47,11 @@ export const useMozillaAuth = (): {
     authService,
     parsedIdToken,
     jwtIdToken,
-    //TODO: Setup mozillians.org groups
+    // Everyone at Pocket has read-only access to this tool
+    // Access groups are checked on the backend when actions (mutations)
+    // are performed.
     canAccessCollections: true, //parsedIdToken.groups.includes('asd'),
     canAccessCuration: true, //parsedIdToken.groups.includes('asd'),
+    canAccessModeration: true,
```
Comment (nina-py) [inline src/_shared/hooks/useMozillaAuth/useMozillaAuth.ts:55]: Clarified the comments here as it's ok for us to leave the frontend as is.

File: src/curated-corpus/pages/CuratedCorpusLandingPage/CuratedCorpusLandingPage.tsx [hunk 2]
```diff
@@ -55,7 +55,7 @@ export const CuratedCorpusLandingPage = (): JSX.Element => {
           <Route exact path={`${path}/`}>
             <h2>Welcome to Pocket&apos;s Curated Corpus management tool.</h2>
 
-            <p>Use the menu above to nagivate:</p>
+            <p>Use the menu above to navigate:</p>
```
Comment (nina-py) [inline src/curated-corpus/pages/CuratedCorpusLandingPage/CuratedCorpusLandingPage.tsx:58]: Unrelated page - fixing a typo

File: src/moderation/pages/ModerationLandingPage/ModerationLandingPage.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                </ListItemText>
+              </ListItem>
+            </List>
+          </Route>
+          {/*<Route exact path={`${path}/search/`}>*/}
+          {/*  <SearchShareableListsPage />*/}
+          {/*</Route>*/}
+          {/*<Route exact path={`${path}/users/`}>*/}
+          {/*  <ManagePilotUsersPage />*/}
+          {/*</Route>*/}
```
Comment (nina-py) [inline src/moderation/pages/ModerationLandingPage/ModerationLandingPage.tsx:65]: Note I commented out these routes as the pages don't yet exist.

File: src/_shared/pages/LandingPage/LandingPage.tsx [hunk 4]
```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import { Grid, Paper, Typography } from '@mui/material';
 import CollectionsBookmarkIcon from '@mui/icons-material/CollectionsBookmark';
+import GavelIcon from '@mui/icons-material/Gavel';
```
Comment (jpetto) [inline src/_shared/pages/LandingPage/LandingPage.tsx:4]: nice 😆

**Review by jpetto — APPROVED** [review]
lgtm!

**Pull request merged**